Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -500,6 +509,15 @@ define([
             }
         }
 
+        var clippingPlanes = this.clippingPlanes;
+        if (defined(clippingPlanes) && clippingPlanes.enabled) {
+            var planeIntersection = clippingPlanes.computeIntersectionWithBoundingVolume(boundingVolume);
+            tile.isClipped = (planeIntersection !== Intersect.INSIDE);
+            if (planeIntersection === Intersect.OUTSIDE) {
+                return Intersect.OUTSIDE;
```

Comment:
The doc for this function says it returns a `Visibility` so this return value doesn't seem correct